aggregate_test.go

```diff
@@ -724,6 +724,2342 @@ func Test_Aggregate(t *testing.T) {
+							Name:    "coredns",
+							Version: "1.0.5"},
+						Component{
+							Name:    "kubernetes",
+							Version: "1.9.2"},
+						Component{
+							Name:    "nginx-ingress-controller",
+							Version: "0.10.2"},
+					},
+					Dependencies: []Dependency{},
+					Deprecated:   true,
+					Name:         "kvm-operator",
+					Time:         time.Date(2018, time.February, 20, 2, 57, 0, 0, time.UTC),
+					Version:      "2.0.1",
+					WIP:          false,
+				},
+				Bundle{
+					Changelogs: []Changelog{
+						Changelog{
+							Component:   "kvm-node-controller",
+							Description: "Updated KVM node controller with pod status bugfix.",
+							Kind:        "changed"},
+						Changelog{
+							Component:   "Calico",
+							Description: "Updated to 3.0.2.",
+							Kind:        "changed"},
+						Changelog{
+							Component:   "kubelet",
+							Description: "Tune kubelet flags for protecting key units (kubelet and container runtime) from workload overloads.",
+							Kind:        "changed"},
+						Changelog{
+							Component:   "etcd",
+							Description: "Updated to 3.3.1.",
+							Kind:        "changed"},
+						Changelog{
+							Component:   "qemu",
+							Description: "Fixed formula for calculating qemu memory overhead.",
+							Kind:        "fixed"},
+						Changelog{
+							Component:   "monitoring",
+							Description: "Added configuration for monitoring endpoint IP addresses.",
+							Kind:        "added"},
+						Changelog{
+							Component:   "cloudconfig",
+							Description: "Enable aggregation layer to be able to extend kubernetes API.",
+							Kind:        "changed"},
+					},
+					Components: []Component{
+						Component{
+							Name:    "calico",
+							Version: "3.0.2"},
+						Component{
+							Name:    "containerlinux",
+							Version: "1576.5.0"},
+						Component{
+							Name:    "docker",
+							Version: "17.09.0"},
+						Component{
+							Name:    "etcd",
+							Version: "3.3.1"},
+						Component{
+							Name:    "coredns",
+							Version: "1.0.6"},
+						Component{
+							Name:    "kubernetes",
+							Version: "1.9.2"},
+						Component{
+							Name:    "nginx-ingress-controller",
+							Version: "0.11.0"},
+					},
+					Dependencies: []Dependency{},
+					Deprecated:   true,
+					Name:         "kvm-operator",
+					Time:         time.Date(2018, time.March, 1, 2, 57, 0, 0, time.UTC),
+					Version:      "2.1.0",
+					WIP:          false,
+				},
+				Bundle{
+					Changelogs: []Changelog{
+						Changelog{
+							Component:   "containerlinux",
+							Description: "Updated to version 1632.3.0.",
+							Kind:        "changed"},
+						Changelog{
+							Component:   "cloudconfig",
+							Description: "Removed set-ownership-etcd-data-dir.service.",
+							Kind:        "changed"},
+					},
+					Components: []Component{
+						Component{
+							Name:    "calico",
+							Version: "3.0.2"},
+						Component{
+							Name:    "containerlinux",
+							Version: "1632.3.0"},
```

Comment:
I would like to have a normal formatting.

Reply:
IMO `gofmt` "should" do that. We should maybe check with the golang wolfpack what they think. Maybe we can get this better. Then I think you need some bash magic search and replace or whatever your editor/IDE is capable of.